tutorpicasso/commands/run_extra_commands.py

```diff
@@ -0,0 +1,142 @@
+    )
+    config: Any = tutor_config.load(tutor_root)
+    picasso_extra_commands: Any = config.get("PICASSO_EXTRA_COMMANDS", None)
+    if picasso_extra_commands is not None:
+        validate_commands(picasso_extra_commands)
+        for command in picasso_extra_commands:
+            run_command(command)
+
+
+def validate_commands(commands: Any) -> None:
+    """
+    Takes all the extra commands sent through config.yml and verifies that
+    all the commands are correct before executing them
+
+    Args:
+        commands (list[str] | None): The commands sent through PICASSO_EXTRA_COMMANDS in config.yml
+    """
+    splitted_commands = [
+        split_string(command, COMMAND_CHAINING_OPERATORS) for command in commands
+    ]
+    flat_commands_array: List[str] = sum(splitted_commands, [])
+
+    invalid_commands = []
+    misspelled_commands = []
+    for command in flat_commands_array:
+        if "tutor" not in command.lower():
+            if find_tutor_misspelled(command):
+                misspelled_commands.append(command)
+            else:
+                invalid_commands.append(command)
+
+        if invalid_commands or misspelled_commands:
+            error_message = (
+                f"Found some issues with the commands:\n\n"
+                f"{'=> Invalid commands: ' if invalid_commands else ''}"
+                f"{', '.join(invalid_commands) if invalid_commands else ''}\n"
+                f"{'=> Misspelled commands: ' if misspelled_commands else ''}"
+                f"{', '.join(misspelled_commands) if misspelled_commands else ''}\n"
+                f"Take a look at the official Tutor commands: "
+                f"https://docs.tutor.edly.io/reference/cli/index.html"
+            )
+            raise click.ClickException(error_message)
+
+
+def run_command(command: str) -> None:
+    """
+    Run an extra command.
+
+    This method runs the extra command provided.
+
+    Args:
+            command (str): Tutor command.
+    """
+    try:
+        with subprocess.Popen(
+            command,
+            shell=True,
+            executable="/bin/bash",
+            stdin=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            text=True,
+        ) as process:
+
+            # It is sent a 'y' to say 'yes' on overriding the existing folders
+            stdout, stderr = process.communicate(input="y")
+
+            if process.returncode != 0 or "error" in stderr.lower():
+                raise subprocess.CalledProcessError(
+                    process.returncode, command, output=stdout, stderr=stderr
+                )
+
+            # This print is left on purpose to show the command output
+            click.echo(stdout)
+
+    except subprocess.CalledProcessError as error:
+        raise click.ClickException(str(error))
+
+
+def find_tutor_misspelled(command: str) -> bool:
+    """
+    This function takes a command and looks if it has the word 'tutor' misspelled
```

Comment:
```suggestion
    Look for misspelled occurrences of the word `tutor` in a given string. E.g. ...
    
    Args:
        command (str): string to be reviewed.
    
    Return:
        True if any misspelled occurrence is found, False otherwise.
```